src/ngx_http_lua_accessby.c

```diff
@@ -374,4 +373,20 @@ ngx_http_lua_access_by_chunk(lua_State *L, ngx_http_request_t *r)
     return NGX_DECLINED;
 }
 
+static ngx_int_t
+ngx_http_lua_access_by_chunk(lua_State *L, ngx_http_request_t *r)
```

Comment:
Never mind, the problem I ran into before is not about lua-nginx-module handler do not explictly discard request body; it is about Nginx itself does not discard the request body in the presence of custom error handler.